modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -572,36 +593,149 @@ private long _getRootObjectEntryAccountEntryId(
 			accountEntryRestrictedObjectField.getName());
 	}
 
+	private Date _getStartDate() {
+		return Date.from(
+			LocalDate.now(
+			).minusDays(
+				Objects.equals(_objectConfiguration.timeScale(), "days") ?
+					_objectConfiguration.duration() :
+						_objectConfiguration.duration() * 7
+			).atStartOfDay(
+				ZoneId.systemDefault()
+			).toInstant());
+	}
+
+	private void _sendNotificationToAdminUsers(
```

Comment:
I think we can rename this method to `_sendNotificationToInstanceAdministrators`

Reply:
Did you have a chance to rename this method?